.cirrus.yml

```diff
@@ -0,0 +1,58 @@
+task:
+  trigger_type: manual
+  name: E2E/Integration Tests
+  timeout_in: 45
+  container:
+    image: ubuntu:20.04
+    kvm: true # required for E2E/Integrations tests
+    cpu: 6
+    memory: 12G
+  env:
+    DEBIAN_FRONTEND: noninteractive
+    CI: true
+    CIRRUS_CLONE_DEPTH: 0 # Required to get app version
+    FORCE_COLOR: 1 # force terminal color
+    PLAYWRIGHT_SKIP_BROWSER_DOWNLOAD: 1 # disable auto-browser download
+    KEYRING: /usr/share/keyrings/nodesource.gpg
+    NODE_VERSION: node_14.x # set node version e.g: "node_16.x" for Node 16 LTS
+
+  # Setting up a non-root user
+  prepare_user_script:
+    - groupadd --gid $(stat -c '%g' /dev/kvm) kvm
+    - useradd --create-home --groups kvm ranchertest
+    - tar c --owner=ranchertest . | tar x --directory=/home/ranchertest
+    - chown -R ranchertest:ranchertest /home/ranchertest
+
+  install_deps_script: |
+    apt-get update
+    apt-get install -y --no-install-recommends ca-certificates gcc g++ curl git golang openssh-client make netcat sudo vim gpg lsb-core \
+                    libgtk2.0-0 libgtk-3-0 libgbm-dev libnotify-dev libgconf-2-4 libnss3 libxss1 libasound2 libxtst6 xauth xvfb
+    export DISTRO="$(lsb_release --short --codename)"
+    curl --silent --location --fail --output /dev/null "https://deb.nodesource.com/${NODE_VERSION}/dists/${DISTRO}/Release"
```

Comment:
Please add a comment on why we're doing this thing that seems to be a no-op. (I realize you already mentioned this in a PR comment; please commit that to code.)

Reply:
Did a command split in order to be more concise, splitting ubuntu dependencies install  and node+npm install. 
Also provided a 4 lines comments explaining what's each steps.